src/javascript/app/Stores/Modules/Trading/Helpers/active-symbols.js

```diff
@@ -34,3 +30,33 @@ export const isMarketClosed = (active_symbols = [], symbol) => {
         :
         false;
 };
+
+export const pickDefaultSymbol = (active_symbols = []) => {
+    if (!active_symbols.length) return '';
+    return getFavoriteOpenSymbol(active_symbols) || getFirstOpenSymbol(active_symbols);
+};
+
+const getFavoriteOpenSymbol = (active_symbols) => {
+    const chart_favorites = LocalStore.get('cq-favorites');
+    if (!chart_favorites) return undefined;
+    const client_favorite_markets = JSON.parse(chart_favorites)['chartTitle&Comparison'];
+
+    const client_favorite_list = client_favorite_markets
+        .map(client_fav_symbol => active_symbols.find(symbol_info => symbol_info.symbol === client_fav_symbol));
+    if (client_favorite_list) {
+        const client_first_open_symbol = client_favorite_list.filter(symbol => symbol).find(isSymbolOpen);
+        if (client_first_open_symbol) return client_first_open_symbol.symbol;
+    }
+    return undefined;
+};
+
+const getFirstOpenSymbol = (active_symbols) => {
+    const firs_open_symbol = active_symbols.filter(symbol_info => /major_pairs|random_index/.test(symbol_info.submarket))
```

Comment:
- Typo in variable name. 
- Nitpick: Break down the line into multiple ones, it is easier to read.